data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_races.lst

```diff
@@ -34,24 +34,26 @@ Banedead																	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Wa
 Banelar																	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:H	MOVE:Walk,30,Swim,30	FACE:15	REACH:15	VISION:Darkvision (60')|Low-Light Vision													BONUS:COMBAT|AC|5|TYPE=NaturalArmor																																	BONUS:STAT|STR|16	BONUS:STAT|DEX|2	BONUS:STAT|CON|10	BONUS:STAT|INT|6	BONUS:STAT|WIS|6	BONUS:STAT|CHA|6		BONUS:VAR|PetrificationResistanceBonus|3	BONUS:VAR|FastHealing|2																																		MONCSKILL:Concentration|Diplomacy|Intimidate|Spot|Use Magic Device																																					AUTO:LANG|Common|Draconic|Orc																							ABILITY:Special Ability|AUTOMATIC|Fast Healing|Immunity to Acid|Immunity to Poison|Poison|Resistance to Petrification																																	LEGS:0	HANDS:0	NATURALATTACKS:Sting,Weapon.Natural.Melee.Finesseable.Piercing.Poison,*1,2d4|Bite,Weapon.Natural.Finesseable.Melee.Piercing.Slashing.Poison,*1,1d3																						MONSTERCLASS:Banelar:7				RACETYPE:Magical Beast							TYPE:Magical Beast	TEMPLATE:Magical Beast									HITDICEADVANCEMENT:7,15,21					CR:5			SOURCEPAGE:p.
-Bat (Deep/Bonebat)			OUTPUTNAME:Deep Bat, Bonebat								STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Walk,10,Fly,50	FACE:5	REACH:5	VISION:Darkvision (60')							PREALIGN:NE								BONUS:COMBAT|AC|2|TYPE=NaturalArmor																																	BONUS:STAT|STR|2	BONUS:STAT|DEX|2	BONUS:STAT|INT|-4	BONUS:STAT|CHA|4								BONUS:VAR|Maneuverability|3																																								MONCSKILL:Hide|Listen|Move Silently|Spot																																																																					ABILITY:Special Ability|AUTOMATIC|Paralysis (Su)|See Invisibility	ABILITY:Special Ability|AUTOMATIC|Flight Maneuverability																																LEGS:0	HANDS:2	NATURALATTACKS:Bite,Weapon.Natural.Finesseable.Melee.Piercing.Slashing,*1,1d6																													DR:5/Bludgeoning			MONSTERCLASS:Undead:4				RACETYPE:Undead								TYPE:Undead			TEMPLATE:Undead																			CR:2			SOURCEPAGE:p.
+Bat (Deep/Bonebat)			OUTPUTNAME:Deep Bat, Bonebat								STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Walk,10,Fly,50	FACE:5	REACH:5	VISION:Darkvision (60')							PREALIGN:NE								BONUS:COMBAT|AC|2|TYPE=NaturalArmor																																	BONUS:STAT|STR|2	BONUS:STAT|DEX|2	BONUS:STAT|INT|-4	BONUS:STAT|CHA|4								BONUS:VAR|Maneuverability|3																																								MONCSKILL:Hide|Listen|Move Silently|Spot																																																																					ABILITY:Special Ability|AUTOMATIC|Paralysis ~ Bonebat|See Invisibility	ABILITY:Special Ability|AUTOMATIC|Flight Maneuverability																																LEGS:0	HANDS:2	NATURALATTACKS:Bite,Weapon.Natural.Finesseable.Melee.Piercing.Slashing,*1,1d6																													DR:5/Bludgeoning			MONSTERCLASS:Undead:4				RACETYPE:Undead								TYPE:Undead			TEMPLATE:Undead																			CR:2			SOURCEPAGE:p.
```

Comment:
I suspect this is not originally a file I created. Not following current conventions at all...

There should only be one single ability call on each race for easier manipulation - ABILITY:Internal|AUTOMATIC|Race Traits ~ Race Name or something similar.